### ISSUE TITLE
Documentation about using labels to identity k8s objects is misleading

### ISSUE THREAD

**Issue by CarstenLeue** [ibm-hyper-protect/k8s-operator-hpcr#43]
in https://github.com/ibm-hyper-protect/k8s-operator-hpcr/blob/main/Using-OnPrem.md the examples are using explicit labels for the resource for illustration purposes, e.g. `app: onpremtest` or `config:onpremtest`

According to feedback this gives the impression that the labels have to follow a certain pattern, e.g. always `app: XXX`. The data disk example however uses a different pattern `config: YYY`. I did this intentionally to point out that there is no semantic or mandatory pattern. Also `Notice how the selector config: onpremsample selects the SSH configuration.` states that.

However when copy/pasting the example as is it will not work, because the example config has the label `app: onpremtest` and not `config: onpremtest`.

@HenryWelborn @Timo-1 do you have a suggestion how to make this clearer or more consumable?
Personally I am not a big fan of copy/paste since this will not scale for more than one resource. Maybe we should leave out explicit labels altogether? What do you think?

cc @vishwabmc

**Comment by HenryWelborn** [comment]
In https://github.com/ibm-hyper-protect/k8s-operator-hpcr/blob/main/Using-OnCloud.md#planning I was trying to lay out how to plan for labels. Maybe that needs some further clarification and copied into Using-OnPrem.md ?

**Comment by CarstenLeue** [comment]
yes, I think it would be helpful to copy this info over to the onprem version of the article and to make sure the sample value is used consistently. 

One aspect we might probably want to mention is that some of the resources are meant to be shared across different virtual server instances, such as the config map containing the VPC config for cloud or the ssh config for onprem (since there may be many VSIs running on one LinuxONE box and they all can share the same config)

So the label for the VSI does not have to match the label on the configs as suggested by https://github.com/ibm-hyper-protect/k8s-operator-hpcr/blob/main/Using-OnCloud.md#planning. What has to match are the references from the VSI to the configs (or to the data disk in the onprem case).
